src/liballoc/collections/btree/map.rs

```diff
@@ -1662,6 +1661,8 @@ impl<'a, K, V> Range<'a, K, V> {
         self.front == self.back
     }
 
+    /// Unsafe because the caller must ensure that `self.front` is not on the last leaf edge
+    /// in the tree.
```

Comment:
These comments sound plausible, but I cannot tell if they are correct. Cc @rust-lang/libs

Reply:
They're a mechanical mapping from the implementation and the underlying documentation (which has been discussed and merged recently). But if you prefer, I can take them out again.

Reply:
You are referring to the comments at https://github.com/rust-lang/rust/blob/0262b3bfe2abc704513aeafd0be056613d66e213/src/liballoc/collections/btree/navigate.rs#L134?

Reply:
Yes, the fruit of #68468.

Reply:
I'm taking the out because I want them to consider the invariant. Actually make the bridge to the `is_empty` in the body of `clone_from` that sparked this.